br/pkg/lightning/backend/external/util_test.go

```diff
@@ -171,3 +171,40 @@ func TestGetAllFileNames(t *testing.T) {
 		"/subtask/3/0", "/subtask/3/1", "/subtask/3/2",
 	}, dataFiles)
 }
+
+func TestCleanUpFiles(t *testing.T) {
+	ctx := context.Background()
+	store := storage.NewMemStorage()
+	w := NewWriterBuilder().
+		SetMemorySizeLimit(20).
+		SetPropSizeDistance(5).
+		SetPropKeysDistance(3).
+		Build(store, 0, "/subtask")
```

Comment:
```suggestion
		Build(store, "/subtask", 0)
```

I have changed the parameter order in #46387.